README.rst

```diff
@@ -60,85 +66,83 @@ Here's an example of the things needed inside a Scrapy project's ``settings.py``
 
     TWISTED_REACTOR = "twisted.internet.asyncioreactor.AsyncioSelectorReactor"
 
-Usage
------
-
-To enable a ``scrapy.Request`` to go through Zyte API, the ``zyte_api`` key in
-`Request.meta <https://docs.scrapy.org/en/latest/topics/request-response.html#scrapy.http.Request.meta>`_
-must be present and contain a dict with Zyte API parameters:
+The ``ZYTE_API_ENABLED`` setting, which is ``True`` by default, can be set to
+``False`` to disable this plugin.
 
-.. code-block:: python
 
-    import scrapy
+Usage
+=====
```

Comment:
I think there are 2 gotchas which we currently don't call out explicitly. 

1. If you use manual mode with httpResponseBody, you need to pass httpResponseHeaders=true for decoding to work properly. In general, Request parameter mapping is documented very well in the README, but I'm not sure if existing docs are enough for the Response parameter mapping. 
2. In the section about default parameters we should probably mention that using browserHtml=True (or anything other than httpResponseBody + httpResponseHeaders as the main output types) for all requests is probably a very bad idea, because it'd be applied for robots.txt, sitemap and other similar requests; it's not going to work correctly.

What do you think about adding them somewhere to the document?

Reply:
Response mapping now has its own section with 2 examples and detailed parameter mapping. Also covered 2.